app/src/main/java/org/wildaid/ofish/ui/home/HomeFragment.kt

```diff
@@ -133,6 +142,8 @@ class HomeFragment : Fragment(R.layout.fragment_home),
             this.homeActivityViewModel = this@HomeFragment.activityViewModel
         }
 
+        fragmentViewModel.findAmountOfDrafts()
```

Comment:
Fragment should not call this method.. 
View model should start fetching info in init block